internal/githubv4mock/githubv4mock.go

```diff
@@ -0,0 +1,235 @@
+// githubv4mock package provides a mock GraphQL server used for testing queries produced via
```

Comment:
This whole package should have its own tests but I want to get feedback on it before the extra effort.